src/App.js

```diff
@@ -98,7 +98,7 @@ const App = (props) => {
           execute: executePermission,
         },
         isReceptorConfigured,
-        hasSmartManagement,
+        hasSatellite,
```

Comment:
I would prefer avoiding changing internal variables since this change will, for example, break components here https://github.com/RedHatInsights/insights-remediations-frontend/blob/9dfb812ffcb1088d8201ea1be50063cbea35e2f6/src/components/PlaybookCard.js#L80 and other. Given that this is a context variable, you have to make sure all consumer components talk to the correct variable. However, we can ignore it and only change the client facing texts :-)

Reply:
Smart management gates are removed. RHIF-245, we are safe here as of yesterday.